src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs

```diff
@@ -814,7 +814,9 @@ private void UpdateCheckboxVisibility(object sender, bool isPointerOver)
 			if (sender is ListViewItem control && control.FindDescendant<UserControl>() is UserControl userControl)
 			{
 				// Handle visual states
```

Comment:
Can you update the comment to include the requirements for the feature?

Reply:
Added a comment.